networking/network_observability/network-observability-overview.adoc

```diff
@@ -0,0 +1,47 @@
+:_content-type: ASSEMBLY
+[id='network-observability-overview']
+= About Network Observability
+include::_attributes/common-attributes.adoc[]
+:context: network-observability-overview
+
+toc::[]
+
+Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters.
+
+[id="dependencies-network-observability"]
+== Dependencies of Network Observability Operator
+
+The Network Observability Operator requires the following Operators:
+
+* Loki Operator: You must install Loki install before you install the Network Observability Operator. The Loki Operator is the backend that you can use to store all collected flows.
+* Grafana Operator: You can optionally install Grafana Operator for using custom dashboards and querying capabilities.
+* Kafka Operator: You can optionally install Kafka Operator. The Kafka Operator provides scalability, resiliency and high availability in the {product-title} cluster.
```

Comment:
```suggestion
* Red Hat Integration - AMQ Streams: You can optionally install Kafka using AMQ Streams. This Operator provides scalability, resiliency and high availability in the {product-title} cluster.
```

Reply:
`Red Hat Integration - AMQ Streams` is downstream name
`Strimzi` is the upstream one

Both are Kafka operators for OCP

Reply:
Addressed- added installing kafka operator using AMQ streams as a recommendation